source/enable-desktops/modify-form-attributes.rst

```diff
@@ -280,5 +280,36 @@ value of ``8`` in the "Number of hours" form field.
    when you go back to the form page for that given app, it will auto-fill in
    the form with your previous values.
 
+
+Minimal LinuxHost Form
+----------------------
+
+Because the :ref:`resource-manager-linuxhost` is not like a traditional scheduler,
+there are very few form items you'll need to add or have your users choose from.
+
+This is a minimal configuration to launch an XFCE desktop environment on a cluster
+we call owens_login.  You'll notice a lot of entries are null because they don't
+really have any meaning in the LinuxHost Adapter.
+
+.. code-block:: yaml
+   :emphasize-lines: 5-
+
+   # /etc/ood/config/apps/bc_desktop/owens_login_desktop.yml
+   ---
+   title: "Owens Login XFCE desktop"
+   description: |
+      This launches a XFCE desktop on an Owens login nodes.
+   cluster: "owens_login"
+   attributes:
+   desktop: "xfce"
+   bc_num_hours:
+      value: 1
+   bc_num_slots: null
+   bc_queue: null
+   bc_account: null
+   node_type: null
+   bc_email_on_started: null
```

Comment:
the attributes need indented underneath `attributes:` section i.e.

```
attributes:
  bc_num_hours:
    value: 1
  bc_num_slots: null
```

I wonder if a replacement of the `form:` section itself is preferable to specifying null for all the unused fields. I think if you do this you get the same outcome:

```yaml
---
title: Owens Login XFCE desktop
description: This launches a XFCE desktop on an Owens login nodes.
cluster: owens_login
form:
  - desktop
  - bc_num_hours
attributes:
  bc_num_hours:
    value: 1
  desktop: "xfce"
```

I do not know which is preferable to recommend.

Reply:
Yea I think specifying the form as you have it is a lot cleaner. Updated for the same.